src/tests/debug-tests.c

```diff
@@ -190,50 +190,50 @@ int test_helper_debug_check_message(int level)
     msg[fsize] = '\0';
 
     if (debug_timestamps == 1) {
-        char time_day[4] = {'\0', '\0', '\0', '\0'};
-        char time_month[4] = {'\0', '\0', '\0', '\0'};
-        int time_day_num = 0;
         int time_hour = 0;
         int time_min = 0;
         int time_sec = 0;
         int time_usec = 0;
+        int time_day = 0;
+        int time_month = 0;
         int time_year = 0;
         int scan_return = 0;
 
         if (debug_microseconds == 0) {
-            scan_return = sscanf(msg, "(%s %s %d %d:%d:%d %d)", time_day, time_month,
-                                 &time_day_num, &time_hour, &time_min, &time_sec, &time_year);
+            scan_return = sscanf(msg, "(%d.%d.%d %d:%d:%d)", &time_year,
+                                 &time_month, &time_day, &time_hour,
```

Comment:
IMHO YYYY-MM-DD would be better as dash is usually used as separator in this format.

Reply:
Ok, I will update the patch.

Reply:
Done.